tests/run-make/incr-add-rust-src-component/rmake.rs

```diff
@@ -0,0 +1,39 @@
+// rust-lang/rust#70924: Test that if we add rust-src component in between
+// two incremental compiles, the compiler does not ICE on the second.
+// Remove the rust-src part of the sysroot for the *first* build.
+// Then put in a facsimile of the rust-src
```

Comment:
Suggestion: `facsimilie` -> `copy`. As a non-native English speaker, I had to google what this fancy word meant 😆

Reply:
Hey, for once, it wasn't me who wrote this part!